tests/unit/net_test.ts

```diff
@@ -1277,6 +1277,26 @@ Deno.test({
   }
 });
 
+Deno.test(
+  { permissions: { net: true } },
+  async function netTcpWithAbortSignal() {
+    const controller = new AbortController();
+    setTimeout(() => controller.abort(), 1000);
```

Comment:
I think 100ms is enough.

```suggestion
    setTimeout(() => controller.abort(), 100);
```